website/tests/test_cve_cache.py

```
@@ -0,0 +1,448 @@
+        """Test IndexError handling when accessing list index fails."""
+        mock_response = Mock()
+        # Create a scenario where accessing [0] raises IndexError
+        # We need to mock a list-like object that passes len() but fails on [0]
+        from unittest.mock import MagicMock
+        
+        class IndexErrorList:
+            def __len__(self):
+                return 1  # Passes len() check
+            
+            def __getitem__(self, key):
+                raise IndexError("list index out of range")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)